alerts/alert.go

```diff
@@ -0,0 +1,71 @@
+package alerts
+
+import "github.com/MohawkTSDB/mohawk/backend"
+
+const (
+	CLOSED_INTERVAL RangeIntervalType = iota
```

Comment:
I like better:
HIGHER_THEN
LOWER_THEN
BETWEEN

I want mohawk to be simple also in respect to names, and lower then sounds to me simpler the left closed whatever :-)

Reply:
👍 😸